Required for vala-0.10.2; already committed upstream.

--- origsrc/synapse-0.2.2.2/src/tile-view/tile.vala	2010-12-06 14:55:46.000000000 -0600
+++ src/synapse-0.2.2.2/src/tile-view/tile.vala	2011-01-20 01:27:16.646868500 -0600
@@ -206,11 +206,7 @@ namespace UI.Widgets
       {
         try
         {
-#if VALA_0_12
           Gdk.Pixbuf temp_pb;
-#else
-          unowned Gdk.Pixbuf temp_pb;
-#endif
           unowned IconTheme it = IconTheme.get_default ();
           try
           {
@@ -225,10 +221,6 @@ namespace UI.Widgets
                                     IconLookupFlags.FORCE_SIZE);
           }
           pixbuf = temp_pb.copy ();
-#if VALA_0_12
-#else
-          temp_pb.unref (); // careful here!
-#endif
         }
         catch (Error err)
         {
--- origsrc/synapse-0.2.2.2/src/ui-widgets.vala	2010-12-14 14:10:47.000000000 -0600
+++ src/synapse-0.2.2.2/src/ui-widgets.vala	2011-01-20 01:26:54.852621900 -0600
@@ -1703,11 +1703,7 @@ namespace Synapse.Gui
       bool changed = false;
       var context = this.get_layout ();
       var attrs = context.get_attributes ();
-#if VALA_0_12
       Pango.AttrIterator iter = attrs.get_iterator ();
-#else
-      unowned Pango.AttrIterator iter = attrs.get_iterator (); // FIXME: leaks
-#endif
       do
       {
         unowned Pango.Attribute? attr = iter.get (Pango.AttrType.SCALE);
